stock_available/__openerp__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'Stock available to promise',
-    'version': '9.0.1.0.0',
+    'version': '9.0.3.0.0',
```

Comment:
https://github.com/OCA/maintainer-tools/compare/master...clonedagain:patch-3 makes it clearer